pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -145,6 +147,7 @@ func (r *ReconcileStorageCluster) ensureExternalStorageClusterResources(instance
 		Kind:       instance.Kind,
 		Name:       instance.Name,
 	}
+	var extCephObjectStores []*cephv1.CephObjectStore
```

Comment:
Why an array?

Reply:
This is to pass it to `createCephObjectStoreInstances` method (which takes an array of `*cephv1.CephObjectStore` instances as first argument).